spec/furniture/marketplace/tag_spec.rb

```diff
@@ -1,5 +1,7 @@
 require "rails_helper"
 
 RSpec.describe Marketplace::Tag, type: :model do
-  it { is_expected.to validate_uniqueness_of(:label).case_insensitive.scoped_to(:bazaar_id) }
+  subject { build(:marketplace_tag) }
```

Comment:
This test started complaining because tags were being created without a marketplace_id foreign key. Not sure why it worked for bazaar before?

Reply:
The relationship to `bazaar` did not have a non-null constraint, but the new `marketplace` relationship does.

Reply:
Ah yes, of course. Free floating tags.